app/experimenter/nimbus-ui/src/components/PageRequestReview/DraftStatusOperations.tsx

```diff
@@ -0,0 +1,162 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Not exactly part of your PR, but what do you think about using directories for components in `PageRequestReview`? We do this in `PageResults`. I think if those components need their own test file and possible story file, it'd help with organization... I went ahead and created a directory in my reject feedback PR since I included a test file, but don't want that to be the only dir in `PageRequestReview`.

Reply:
We didn't want to nest too many directories deep, so I started going for just single files for these little components.

They're mostly presentational components or utilities for the page. I don't think they should have much on their own to test or exercise in stories. Most of it should happen in conjunction with the page using them.

Reply:
Also FWIW, I was holding off on tests for these components until #1062 because they could end up being changed or juggled around a bit once we start actually trying to integrate with the backend

Reply:
Yeah, that seems fine. I can pull mine out when I edit my PR after this one gets merged and make it more in tune with what you've got here.